compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -213,6 +213,11 @@ def AMDAIEHoistForLoopAffineApply : Pass<"iree-amdaie-hoist-for-affine-apply"> {
   let constructor = "mlir::iree_compiler::AMDAIE::createAMDAIEHoistForLoopAffineApplyPass()";
 }
 
+def AMDAIEHoistLogicalObjFifo : Pass<"iree-amdaie-hoist-logical-objectfifo"> {
+  let summary = "Hoist logical objectFifo operations to the scope of its operands.";
```

Comment:
```suggestion
  let summary = "Hoist logical objectFifo operations to the scope of a most nested operand, without hoisting through workgroup, controlcode, or func ops.";
```

Reply:
I guess it makes sense to not hoist through these ops because they're isolated from above (could you check that instead?)

Reply:
Scrap that idea, they're not isolated from above